### PR TITLE
[studio-ui] SearchPanel and SearchPage updates

### PR DESCRIPTION
**Pull request by aachavarria**
SearchPage: DarkTheme Fixes;
PathSelector fixing issues;
SearchPanel: Fixing issue with Pagination and adding loading state;

### CODE REVIEW

File: ui/app/src/modules/Search/FilterSearchDropdown.tsx [hunk 1]
```diff
@@ -134,7 +137,7 @@ const useStyles = makeStyles((theme: Theme) => ({
   },
   pathSelectorWrapper: {
     display: 'flex',
-    background: palette.white,
+    // background: palette.white,
```
Comment (rart) [inline ui/app/src/modules/Search/FilterSearchDropdown.tsx]: x

File: ui/app/src/modules/Search/FilterSearchDropdown.tsx [hunk 2]
```diff
@@ -828,9 +831,9 @@ export default function FilterSearchDropdown(props: FilterSearchDropdownProps) {
         }}
       >
         <List classes={{ padding: classes.listPadding }}>
-          <div>
+          <>
```
Comment (rart) [inline ui/app/src/modules/Search/FilterSearchDropdown.tsx]: What’s the fragment doing here? Same goes for the one next too.

**Review by rart — CHANGES_REQUESTED** [review]

File: ui/app/src/modules/Search/FilterSearchDropdown.tsx (hunk 2 above)
Comment (aachavarria) [reply ui/app/src/modules/Search/FilterSearchDropdown.tsx]: nothing

**Review by rart — APPROVED** [review]

**Comment by rart** [comment]
Will be merged on https://github.com/craftercms/studio-ui/pull/2041

**Pull request closed without merging**